aleph/model/entity.py

```diff
@@ -71,8 +75,9 @@ def to_proxy(self):
             'schema': self.schema,
             'properties': self.data
         })
-        proxy.add('name', self.name)
-        proxy.set('indexUpdatedAt', self.updated_at)
+        if proxy.schema.is_a('Thing'):
+            proxy.add('name', self.name)
+            proxy.set('indexUpdatedAt', self.updated_at)
```

Comment:
I think there are more entity types with indexUpdatedAt. Let's just pass quiet=True to avoid errors.

Reply:
The problem I was trying to avoid is that `name` is not a property for entities that are not `Things`. Added a check to see if `name` is defined first. Or may be I should check if the `Schema` has a `name` property? What's the best way to do that in FtM?

Reply:
Just adding `, quiet=True` to the add and set calls will make it set the value iif the prop exists.